### PR TITLE
fix some warnings

### PR DESCRIPTION
**Pull request by elharo**
@michael-o

### CODE REVIEW

File: doxia-site-renderer/src/test/java/org/apache/maven/doxia/siterenderer/DefaultSiteRendererTest.java [hunk 1]
```diff
@@ -233,7 +234,10 @@ public void testVelocityToolManager()
         renderer.mergeDocumentIntoSite( writer, sink, siteRenderingContext );
 
         String renderResult = writer.toString();
-        String expectedResult = IOUtils.toString( getClass().getResourceAsStream( "velocity-toolmanager.expected.txt" ) );
+        String expectedResult =
+            IOUtils.toString(
+                getClass().getResourceAsStream("velocity-toolmanager.expected.txt"),
+                StandardCharsets.UTF_8);
```
Comment (michael-o) [inline doxia-site-renderer/src/test/java/org/apache/maven/doxia/siterenderer/DefaultSiteRendererTest.java]: Space missing

File: doxia-site-renderer/src/test/java/org/apache/maven/doxia/siterenderer/AbstractVerifier.java [hunk 2]
```diff
@@ -51,10 +51,11 @@ protected HtmlPage htmlPage( String htmlFile )
         assertTrue( file.exists() );
 
         // HtmlUnit
-        WebClient webClient = new WebClient();
-        webClient.getOptions().setCssEnabled( false );
-
-        return (HtmlPage) webClient.getPage( file.toURI().toURL() );
+        try (WebClient webClient = new WebClient()) {
```
Comment (michael-o) [inline doxia-site-renderer/src/test/java/org/apache/maven/doxia/siterenderer/AbstractVerifier.java]: Spaces missing

File: doxia-site-renderer/src/test/java/org/apache/maven/doxia/siterenderer/DefaultSiteRendererTest.java [hunk 3]
```diff
@@ -290,8 +294,10 @@ public void testVelocityToolManagerForSkin()
         SiteRendererSink sink = new SiteRendererSink( context );
         renderer.mergeDocumentIntoSite( writer, sink, siteRenderingContext );
         String renderResult = writer.toString();
-        String expectedResult = IOUtils.toString( getClass().getResourceAsStream( "velocity-toolmanager.expected.txt" ) );
-        expectedResult = StringUtils.unifyLineSeparators( expectedResult );
+        String expectedResult = StringUtils.unifyLineSeparators(
+            IOUtils.toString(
+                getClass().getResourceAsStream("velocity-toolmanager.expected.txt"),
+                StandardCharsets.UTF_8));
```
Comment (michael-o) [inline doxia-site-renderer/src/test/java/org/apache/maven/doxia/siterenderer/DefaultSiteRendererTest.java]: here too

File: doxia-site-renderer/src/test/java/org/apache/maven/doxia/siterenderer/JavascriptVerifier.java [hunk 4]
```diff
@@ -60,46 +60,47 @@ public void verify( String file )
...
-        assertNotNull( p );
-        assertEquals( "You should see a JavaScript alert...", p.asText().trim() );
-
-        HtmlScript script = (HtmlScript) elementIterator.next();
-        assertNotNull( script  );
-        assertEquals( "text/javascript", script.getAttribute( "type" ) );
-        assertEquals( "", script.asText().trim() );
-        final List<String> expectedAlerts = Collections.singletonList( "Hello!" );
-        assertEquals( expectedAlerts, collectedAlerts );
+        try (WebClient webClient = new WebClient()) {
```
Comment (michael-o) [inline doxia-site-renderer/src/test/java/org/apache/maven/doxia/siterenderer/JavascriptVerifier.java]: same here

File: doxia-site-renderer/src/test/java/org/apache/maven/doxia/siterenderer/AbstractVerifier.java (hunk 2 above)
Comment (elharo) [reply doxia-site-renderer/src/test/java/org/apache/maven/doxia/siterenderer/AbstractVerifier.java]: done

File: doxia-site-renderer/src/test/java/org/apache/maven/doxia/siterenderer/DefaultSiteRendererTest.java (hunk 1 above)
Comment (elharo) [reply doxia-site-renderer/src/test/java/org/apache/maven/doxia/siterenderer/DefaultSiteRendererTest.java]: done

File: doxia-site-renderer/src/test/java/org/apache/maven/doxia/siterenderer/DefaultSiteRendererTest.java (hunk 3 above)
Comment (elharo) [reply doxia-site-renderer/src/test/java/org/apache/maven/doxia/siterenderer/DefaultSiteRendererTest.java]: done

File: doxia-site-renderer/src/test/java/org/apache/maven/doxia/siterenderer/JavascriptVerifier.java (hunk 4 above)
Comment (elharo) [reply doxia-site-renderer/src/test/java/org/apache/maven/doxia/siterenderer/JavascriptVerifier.java]: done

File: doxia-site-renderer/src/test/java/org/apache/maven/doxia/siterenderer/DefaultSiteRendererTest.java (hunk 3 above)
Comment (michael-o) [reply doxia-site-renderer/src/test/java/org/apache/maven/doxia/siterenderer/DefaultSiteRendererTest.java]: Still spaces missing after `UTF_8`.

File: doxia-site-renderer/src/test/java/org/apache/maven/doxia/siterenderer/DefaultSiteRendererTest.java [hunk 5]
```diff
@@ -297,7 +297,7 @@ public void testVelocityToolManagerForSkin()
         String expectedResult = StringUtils.unifyLineSeparators(
             IOUtils.toString(
                 getClass().getResourceAsStream( "velocity-toolmanager.expected.txt" ),
-                StandardCharsets.UTF_8));
+                StandardCharsets.UTF_8 ));
```
Comment (michael-o) [inline doxia-site-renderer/src/test/java/org/apache/maven/doxia/siterenderer/DefaultSiteRendererTest.java]: and another one between the parens.

**Pull request merged**